ddk-configuration/checkstyle/suppressions.xml

```diff
@@ -5,6 +5,8 @@
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
-    <suppress files="Messages.java"
+    <suppress files="Messages\.java"
               checks=".*"/>
+    <suppress files=".*Test.*\.java|.*Bug.*\.java"
+              checks="AvoidStarImport|EmptyBlock|IllegalCatch|JavadocMethod|JavadocType|JavadocStyle|MagicNumber|MethodName"/>

```

Comment:
This should rather be configured in the checkstyle test ruleset (by removing modules and tuning modules configuration).  This way unwanted checks are not executed at all while in the config proposed here checks are executed but the produced markers are just suppressed (less efficient and conceptually wrong IMHO).